ideas/xxx-sob-improve-onboarding.md

```diff
@@ -0,0 +1,77 @@
+## Preamble
+
+    Idea: #XX-sob-improve-onboarding
+    Title: OpenBounty Improved Onboarding
+    Status: In progress
+    Created: 2018-04-04
+
+## Summary
+
+Currently users wanting to publish a bounty need to go through various manual steps, including us whitelisting them for using the platform. This step forces users to wait for us and might make them drop out of the process even if they were committed to create a bounty.
+
+This swarm aims to remove the need for whitelisting users and revamp the onboarding flow at the same time.
+
+## Swarm Participants
+
+- Lead Contributor: @martinklepsch (10hr for MVP, 30hr+ from MVP onwards)
+- Testing & Evaluation: @Churik
+- UX Contributor: @EugeOrtiz 
+- Dev Contributor: @siphuel or @rcullito
+- Contracts Contributor: @3esmit
+
+## Product Description
+
+Currently all users wanting to issue a bounty need to go through a whitelisting process. This is required because we currently pay the gas for deploying the bounty contract. In order to make it easier for users to get started we should remove this whitelisting requirement, allowing users to onboard at their own speed.
+
+There are multiple ways the need for whitelisting could be removed:
+
+- Allow (or force) users to pay gas for contract deployment themselves
+- Pay gas for new users but implement some rate-limiting or similar guards to protect from bad actors
+
+These and other alternatives will be evaluated as part of this swarm, whatever decision is taken will be documented in a [Decision Record](https://github.com/status-im/open-bounty/blob/develop/doc/decisions/0001-record-decisions.md). 
+
+## Iterations
+
+### Iteration 1 — Concept/Design (MVP) `2018-04-20`
+
+- Design (wireframes) onboarding and bounty-issuing flow that properly accounts for the removed need to get whitelisted and guides users through the creation of their first bounty.
+- Decide on the best way to eliminate whitelisting requirement for immediate future.
+
+**Artifacts:**  
+
+- [ ] mockup of onboarding flow for bounty issuers
+- [ ] mockup of onboarding flow for bounty hunters
+- [ ] Updated welcome email text for both
+- [ ] Decision Record for removal of whitelisting requirement
+
```

Comment:
"guides users" just refers to visual guides in the workflow I assume?

Reply:
Well, could be anything UX comes up with. Maybe it's pages, maybe a check list of some kind. Dunno.